src/data-structures/linked-list/__test__/LinkedList.test.js

```diff
@@ -217,4 +217,71 @@ describe('LinkedList', () => {
     expect(node.value.customValue).toBe('test2');
     expect(linkedList.find({ value: 2, customValue: 'test5' })).toBeNull();
   });
+
+  it('should traverse through all nodes of the list from head to tail', () => {
+    const linkedList = new LinkedList();
+
+    linkedList
+      .append(1)
+      .append(2)
+      .append(3);
+
+    const traversedNodes = linkedList.traverse();
```

Comment:
`linkedList.traverse()` should be passed the callback. Also, it should be passed a callback that changes the values, to distinguish it from `toArray()`. e.g. `const callback = (x) => x * 2;`